### PR TITLE
New config ansible-multi-node

### PR DESCRIPTION
**Pull request by tonykay**
##### SUMMARY

New config to replace multiple older configs typically beginning `ans-` and `ansible-`.

Also for new courses or environments using Ansible controller

##### ISSUE TYPE

- New config Pull Request

##### COMPONENT NAME
ansible-multi-node

### CODE REVIEW

File: ansible/configs/ansible-multi-node/sample_vars/aapf-osp-vars.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
```
Comment (miteshget) [inline ansible/configs/ansible-multi-node/sample_vars/aapf-osp-vars.yml:1]: @tonykay, you may wish to change file name from aapf-osp-vars.yml to osp-vars.yml btw this is not mandatory.

File: ansible/configs/ansible-multi-node/sample_vars/sample_vars_ec2.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (miteshget) [inline ansible/configs/ansible-multi-node/sample_vars/sample_vars_ec2.yml:1]: May be you can rename from sample_vars_ec2.yml to ec2_vars.yml

File: ansible/configs/ansible-multi-node/files/cloud_providers/ec2_cloud_template.j2 [hunk 3]
```diff
@@ -0,0 +1,373 @@
+#jinja2: lstrip_blocks: "True"
```
Comment (miteshget) [inline ansible/configs/ansible-multi-node/files/cloud_providers/ec2_cloud_template.j2:1]: @tonykay you may delete this file if not using customized ec2 template just to keep everything aligned with default template.

**Review by miteshget — APPROVED** [review]
Requested file name change are not big deal, hence I am approving the PR.

**Review by prakhar1985 — APPROVED** [review]

**Pull request closed without merging**